Client.py

```diff
@@ -16,14 +16,22 @@ class Client(ConnectionListener):
 
     def __init__(self, host, port):
         self.Connect((host, port))
-        self.player = Player(1, 2, 1)
         print("Client started")
 
     def Network_message(self, data):
         print("got:", data['message'])
         connection.Send(data)
         connection.Pump()
 
+    def Network_playersInfo(self, data):
+        playersDict = data["playersInfo"]["info"]
+        print("Players info: ", playersDict)
+
```

Comment:
What is `playersDict` still doing here ???

Reply:
True, changed to playersInfo